src/main/java/duke/Task.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+    protected int sequence;
```

Comment:
I like your use of variable naming, makes the code more comprehensive more readers

Reply:
Thank you